src/python/gudhi/datasets/remote.py

```diff
@@ -0,0 +1,199 @@
+
+def _fetch_remote(url, filename, dirname = "remote_datasets", file_checksum = None, accept_license = False):
+    """
+    Fetch the wanted dataset from the given url and save it in file_path.
+
+    Parameters
+    ----------
+    url : string
+        The url to fetch the dataset from.
+    filename : string
+        The name to give to downloaded file.
+    dirname : string
+        The directory to save the file to. Default is "remote_datasets".
+    file_checksum : string
+        The file checksum using sha256 to check against the one computed on the downloaded file.
+        Default is 'None'.
+    accept_license : boolean
+        Flag to specify if user accepts the file LICENSE and prevents from printing the corresponding license terms.
+        Default is False.
+
+    Returns
+    -------
+    file_path: string
+        Full path of the created file.
+    """
+
+    file_path = join(dirname, filename)
+
+    # Get the file
+    urlretrieve(url, file_path)
+
+    if file_checksum is not None:
+        checksum = _checksum_sha256(file_path)
+        if file_checksum != checksum:
+            raise IOError("{} has a SHA256 checksum : {}, "
+                        "different from expected : {}."
+                        "The file may be corrupted or the given url may be wrong !".format(file_path, checksum, file_checksum))
+
+    # Print license terms unless accept_license is set to True
+    if not accept_license:
+        license_file = join(dirname, "LICENSE")
+        if exists(license_file) and (file_path != license_file):
+            with open(license_file, 'r') as f:
+                print(f.read())
+
+    return file_path
+
+def fetch_spiral_2d(filename = "spiral_2d.npy", dirname = None):
+    """
+    Fetch "spiral_2d.npy" remotely.
+
+    Parameters
+    ----------
+    filename : string
+        The name to give to downloaded file. Default is "spiral_2d.npy".
+    dirname : string
+        The directory to save the file to. Default is None, meaning that the data home will be set to "~/remote_datasets/spiral_2d".
+
+    Returns
+    -------
+    points: array
+        Array of points stored in "spiral_2d.npy".
+    """
+    file_url = "https://raw.githubusercontent.com/GUDHI/gudhi-data/main/points/spiral_2d/spiral_2d.npy"
+    file_checksum = '88312ffd6df2e2cb2bde9c0e1f962d7d644c6f58dc369c7b377b298dacdc4eaf'
+
+    if dirname is None:
+        dirname = join(get_data_home(dirname), "spiral_2d")
+        makedirs(dirname, exist_ok=True)
+    else:
+        dirname = get_data_home(dirname)
+
+    archive_path = join(dirname, filename)
+
+    if not exists(archive_path):
+        file_path_pkl = _fetch_remote(file_url, filename, dirname, file_checksum)
+
+        return np.load(file_path_pkl, mmap_mode='r')
+    else:
+        return np.load(archive_path, mmap_mode='r')
+
+def fetch_bunny(filename = "bunny.npy", dirname = None, accept_license = False):
+    """
+    Fetch "bunny.npy" remotely and its LICENSE file.
```

Comment:
Hmm, the version we have is the one with 35947 vertices. I didn't find the 5000 one.

Reply:
We used to have `gudhi-devel/data/points/bunny_5000.off`. But the version with 35947 vertices is good. I think it would be useful to mention this number of vertices in the doc.